CONTRIBUTING.md

```diff
@@ -1,86 +1,87 @@
-However, some [formatting rules](doc/dev/CODING_STYLE.md) are not enforced by clang-format and will be checked during the review process.
+F3D continuous integration also checks formatting using clang-format and will inform you if changes needs to be made.
+Some [formatting rules](doc/dev/CODING_STYLE.md) are not enforced by clang-format and will be checked during the review process.
 
-When making changes to the libf3d public API, the CI will warn about making related changes to the bindings. This is required in order to merge the PR.
+When making changes to the libf3d public API, continuous integration will warn about making related changes to the bindings.
+This is required in order to merge the pull request.
 
 ## Changelog
 
 Whenever you make significant changes, either bugfixes or features, please update the [changelog](doc/CHANGELOG.md) "Ongoing development" section.
-There are three subsections. "For F3D Users" are changes that impact actual F3D users and related to F3D behaviors changing. "For developers" are change
-that impact developers that build F3D. "For F3D packagers" are for changes that impact anyone packaging F3D, eg: new CMake options.
+There are three subsections. "For F3D Users" are changes that impact actual F3D users and are related to F3D behaviors changing. "For developers" are changes
+that impact developers that build F3D. "For F3D packagers" are changes that impact anyone packaging F3D, eg: new CMake options.
 
-## F3D architecture
+## F3D Architecture
 
-F3D is separated in different components:
-- The F3D application, in the application folder.
-- The libf3d, in the library folder.
-- The VTKExtensions in the library/VTKExtensions folder.
-- The bindings, python, java and webassembly, in the respective directories.
+F3D is divided into different components:
+- The f3d executable, in the application folder.
+- The libf3d library, in the library folder.
+- The VTKExtensions, in the vtkext folder.
+- The bindings for python, java, and webassembly, in their respective directories.
 - The plugins, providing all the different readers in the plugins directory.
 
-VTKExtensions are separated in different modules.
-- Core, that do not depend on any other VTKExtensions modules are provide services for all modules
-- Readers, that depends on Core and implements many new VTK readers and importers
-- Rendering, that depends on Core and implements the rendering specificities of F3D
-- Applicative, the depends on all other VTKExtension modules and provide services for the libf3d library
+VTKExtensions are divided into different modules:
+- Core, that do not depend on any other VTKExtensions modules, provide services for all modules.
+- Readers, that depend on Core and implement many new VTK readers and importers.
+- Rendering, that depend on Core and implement the rendering specificities of F3D.
+- Applicative, that depend on all other VTKExtension modules and provide services for the libf3d library.
```

Comment:
actually, this is not true anymore.

vtkext contains now only a public module that is documented in PLUGINS.md and a private module that is used exclusively by the libf3d.

Reply:
I had a feeling this was no longer correct when I was not able to find directories or notations about certain modules being classified in the listed ways either in the source or Linux tarball, but wasn't sure, so I did not make a change here. Thanks for clarifying the current status and confirming my suspicions.

Updated the documentation as indicated via commit c1633c7.